docs/concepts-advanced/backups.md

```diff
@@ -1,37 +1,222 @@
+    Using custom settings will override cluster default. Any misconfiguration or
+    invalid/expired credentials may cause retrieval of backups to fail or be
+    inaccessible.
+
+| Environment variable name               | Purpose                                                                                                                                                                |
+|:----------------------------------------|:-----------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| `LAGOON_BAAS_CUSTOM_RESTORE_ENDPOINT`   | Specify the S3 compatible endpoint where any Lagoon initiated restores should be stored. An example for S3 Sydney would be: `https://s3.ap-southeast-2.amazonaws.com`. |
+| `LAGOON_BAAS_CUSTOM_RESTORE_BUCKET`     | Specify the bucket name where any Lagoon initiated restores should be stored.An example custom setting would be: `example-restore-bucket`.                             |
+| `LAGOON_BAAS_CUSTOM_RESTORE_ACCESS_KEY` | Specify the access key Lagoon should use to access the custom restore bucket. An example custom setting would be: `AKIAIOSFODNN7EXAMPLE`.                              |
+| `LAGOON_BAAS_CUSTOM_RESTORE_SECRET_KEY` | Specify the secret key Lagoon should use to access the custom restore bucket. An example custom setting would be: `wJalrXUtnFEMI/K7MDENG/bPxRfiCYEXAMPLEKEY`.          |
+
+The S3 bucket must have public access enabled, as Lagoon will create [presigned
+URLs](https://docs.aws.amazon.com/AmazonS3/latest/userguide/ShareObjectPreSignedURL.html)
+for the objects inside the bucket as needed.
+
+An example AWS IAM policy that you can create to allow access to just the S3
+bucket `example-restore-bucket` is:
+
+```json title="aws_iam_restore_policy.json"
+{
+  "Version": "2012-10-17",
+  "Statement": [
+    {
+      "Effect": "Allow",
+      "Action": [
+        "s3:GetBucketLocation",
+        "s3:ListBucket"
+      ],
+      "Resource": [
+        "arn:aws:s3:::example-restore-bucket"
+      ]
+    },
+    {
+      "Effect": "Allow",
+      "Action": [
+        "s3:PutObject",
+        "s3:GetObject",
+        "s3:GetObjectVersion",
+        "s3:GetBucketLocation",
+        "s3:PutObjectAcl"
+      ],
+      "Resource": [
+         "arn:aws:s3:::example-restore-bucket/*"
+      ]
+    }
+  ]
+}
+```
+
+For increased security and reduced storage costs you can opt into [removing
+restored backups after a set
+lifetime](https://docs.aws.amazon.com/AmazonS3/latest/userguide/object-lifecycle-mgmt.html)
+(e.g. 7 days). Lagoon caters for this scenario gracefully and will re-create any
+restored snapshots as needed.
+
+## Manual Backups
+
+Users can create manual backups by running a backup task, syncing data from one
+environment to another, or by running custom commands in containers.
```

Comment:
I would avoid referring to the task as a `backup` task. Not sure how else to describe it as more than a database dump task?

Reply:
Otherwise we don't have a "manual backup" button (yet?)

Reply:
Ya, that's probably more clear for current state: manual backups aren't "created," they're done/run/performed. So something like:

> Users can manually backup databases and files by running a task, syncing.....

If we add the ability to do on-demand backups from the backups tab, that can be added to the list of manual backup methods.

Reply:
Yah sounds good